mw-webapp/src/utils/auth/writeNewUserData.ts

```diff
@@ -0,0 +1,17 @@
+import {ref, set} from "firebase/database";
+import {db} from "src/firebase";
+
+interface UserData {
+  userId: string
+  userEmail: string | null
+  userName: string | null
+}
+
+export const writeNewUserData = ({userId, userEmail, userName}: UserData) => {
```

Comment:
maybe this const better rename to createNewUser/ addNewUser? Or something like this? What do you think?